### PR TITLE
get_webstat_metrics.py: update ports for neo-exporter

### PR DESCRIPTION
**Pull request by 532910**

### CODE REVIEW

File: get_webstat_metrics.py [hunk 1]
```diff
@@ -54,8 +54,8 @@ async def check_epoch(output, net, rpc_host, netmap_hash):
 
 async def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--url-main", type=str, default='http://localhost:16512/metrics')
-    parser.add_argument("--url-test", type=str, default='http://localhost:16513/metrics')
+    parser.add_argument("--url-main", type=str, default='http://localhost:16513/metrics')
```
Comment (roman-khimov) [inline get_webstat_metrics.py:57]: The default for neo-exporter itself is still 16512, why should we change it?

Comment (532910) [reply get_webstat_metrics.py:57]: 1. I didn't understand the question.
2. This setting is already applied manually if you agree with how https://status.fs.neo.org/ looks like -- just merge this.
3. check neo_exporter play for details

Comment (roman-khimov) [reply get_webstat_metrics.py:57]: 1. https://github.com/nspcc-dev/neo-exporter/blob/c7e72ef1bb0112440af96fdf2908d45e20586d46/config/config.yaml#L17
2. It's a flag, you can set it in any way for production if you'd like to. But default value should match the default in neo-exporter.

Comment (532910) [reply get_webstat_metrics.py:57]: This not a "default value", this is a real setting that is configured in our infra.
If you'd like to talk about defaults start with removing the next line with non-default port `16513`.
I'm turning sami back right now. This patch will be merged or status.fs.neo.org will be broken soon.

**Pull request closed without merging**